imaging-hub.data-commons.org/portal/gitops.json

```diff
@@ -0,0 +1,653 @@
+{
+  "gaTrackingId": "G-WM3JV9YMW3",
+  "ddEnv": "brhprod",
+  "ddUrl": "ddog-gov.com",
+  "graphql": {
+    "boardCounts": [],
+    "chartCounts": [],
+    "projectDetails": "boardCounts"
+  },
+  "components": {
+    "appName": "Biomedical Research Hub",
+    "index": {
+      "introduction": {
+        "heading": "",
+        "text": "The Biomedical Research Hub (BRH) is a standard-based data ecosystem for securely managing, analyzing and sharing biomedical data to support research communities and collaborations."
```

Comment:
this file has several references to BRH - but i thought we were using this https://github.com/uc-cdis/bih-data-commons for the portal config instead? can we just remove this one? i'm a bit confused about how this repo and the `bih-data-commons` repo intersect

Reply:
Thanks! We should update this file as well yeah. I'll see what I can do. 

FEFW is running on `/` and portal is running on `/portal`